include/cpptrace/from_current_macros.hpp

```diff
@@ -0,0 +1,54 @@
+#ifndef CPPTRACE_FROM_CURRENT_MACROS_HPP
```

Comment:
Out of curiosity, is there a sensible way to make either of the following work instead of a new macros header?
```cpp
#include <cpptrace/from_current.hpp>
import cpptrace;
// or
import cpptrace;
#include <cpptrace/from_current.hpp>
```
If a macros header is the best way to do this I think it's good with me - and I'm guessing if so other libraries might follow the same pattern. But I'd love to be able to keep `#include <cpptrace/from_current.hpp>` under modules if possible as that is currently in a sense part of the library interface.